lib/view/resources/additional_resources_view.dart

```diff
@@ -45,12 +45,12 @@ class AdditionalResourcesView extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        backgroundColor: AppColors.appBackground,
+        backgroundColor: AppColors.backgroundPurple,
         appBar: AppBar(
           // Warning brightness interacts with SystemUiOverlayStyle
           // See system_bars.dart comments
           brightness: Brightness.light,
-          backgroundColor: AppColors.appBarBackground,
+          backgroundColor: AppColors.purple50,
```

Comment:
As part of the change, we need to alter the headings to be white:

| Designs in Zeplin  | Screenshot of current app |
| ------------- | ------------- |
| ![Resources : Additional Resources](https://user-images.githubusercontent.com/902252/80576987-747fce00-8a49-11ea-97ba-dca6073efdac.png)   | ![image](https://user-images.githubusercontent.com/902252/80577005-7b0e4580-8a49-11ea-8923-59d74f2c4b83.png)  |

Reply:
+1 on that please! ☝️ 🙏  I know it's an exception to the styles, but it'd really help readability with these screens on top of that purple.